src/Kunstmaan/UtilitiesBundle/Helper/Slugifier.php

```diff
@@ -2,41 +2,22 @@
 
 namespace Kunstmaan\UtilitiesBundle\Helper;
 
+use Behat\Transliterator\Transliterator;
+
 /**
  * Sulgifier is a helper to slugify a certain string
  */
-class Slugifier implements SlugifierInterface
+final class Slugifier implements SlugifierInterface
 {
     /**
      * Slugify a string
      *
-     * @param string $text    Text to slugify
-     * @param string $default Default return value (override when slugify would return an empty string)
+     * @param string $text
      *
      * @return string
      */
-    public function slugify($text, $default = '', $replace = array("'"), $delimiter = '-')
+    public function slugify($text)
```

Comment:
We can keep the delimiter here. The transliterate function https://github.com/Behat/Transliterator/blob/master/src/Behat/Transliterator/Transliterator.php#L431 also has a seperator as argument.